src/store/plugins/usersWebSocketPlugin.js

```diff
@@ -0,0 +1,33 @@
+import { usersEndpoint } from "@/services/API/Endpoints.js";
+
+export default function createUsersWebSocket() {
+  return function connect(store) {
+    var UsersWebsocket = new WebSocket(
+      process.env.VUE_APP_BACKEND_WEBSOCKET + usersEndpoint
+    );
+
+    UsersWebsocket.onopen = () => {
+      console.log("WebSocket is open now.");
+    };
+
+    UsersWebsocket.onmessage = (event) => {
+      let user = JSON.parse(event.data).user;
```

Comment:
this assumes that the socket will always receive data corresponding to a user only